pkg/adapter/apiserver/ref.go

```diff
@@ -67,11 +70,7 @@ func (a *ref) Add(obj interface{}) error {
 		return err
 	}
 
-	if _, err := a.ce.Send(context.Background(), *event); err != nil {
-		a.logger.Info("event delivery failed", zap.Error(err))
-		return err
-	}
-	return nil
+	return a.sendEvent(context.Background(), event, a.reporter)
```

Comment:
no need to pass the reporter as it's in the adapter already

Reply:
correct, fixed it